src/js/gamestate.js

```diff
@@ -606,28 +626,48 @@ class GameState {
   }
 
   faceForCard(card, player) {
-    let opponent = this.localPlayer.name == player.name ? this.remotePlayer : this.localPlayer
+    let opponent = this.opponentForName(player.name)
     opponent.life -= card.attack
-
     card.canAct = false
     player.selectedCard = null;
     this.resolveDamage()
-
     this.showCardDamage(card)
     this.showPlayerDamage(opponent)
-  }
-
-  showCardDamage(card) {
-    // opponent damage animation
-    card.showDamage = true
-    card.damageAnimation = setInterval(() => {
-      card.showDamage = null;
+    let bulletDict = {player, 
+                      startIndex: player.board.indexOf(card)} 
+    this.bullets.push(bulletDict)
+    bulletDict.damageAnimation = setInterval(() => {
+      for (var i =0; i < this.bullets.length; i++) {
+        if (this.bullets[i] === bulletDict) {
+          this.bullets.splice(i,1);
+          break;
+        }
+      }
       window.client.forceUpdate()
```

Comment:
Relying on `forceUpdate` is an anti-pattern. We should be passing new props, which will trigger a re-render.